lib/reek/smells/smell_repository.rb

```diff
@@ -25,6 +25,7 @@ def self.smell_types
       # @return [Array<Reek::Smells::SmellDetector>] All SmellDetectors that we want to filter for
       #         e.g. [Reek::Smells::Attribute].
       def self.eligible_smell_types(filter_by_smells = [])
+        filter_by_smells = Array(filter_by_smells)
```

Comment:
Yes, I'll remove it. It is unfortunate that Ruby's String and Array methods often have the same names.